support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.ts

```diff
@@ -188,24 +186,27 @@ const applyBillingAddressRules = (
 					: `Please enter a ${addressType} state name no longer than 40 characters`,
 			),
 		},
-	]);
+	];
+	return validate(rules);
+};
 
 const applyDeliveryAddressRules = (
 	fulfilmentOption: Option<FulfilmentOptions>,
 	fields: FormFields,
 	addressType: AddressType,
 ): Array<FormError<FormField>> => {
-	const homeRules = validate([
+	const rules: Array<Rule<FormError<FormField>>> = [
```

Comment:
I guess we mostly use the type `T[]` instead of `Array<T>` ? Also I wonder if we really need this type to be so generic? Probably not worth changing here though

Reply:
You can only use that syntax on simple types, otherwise you get this ESLint error `ESLint: Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.(@typescript-eslint/array-type)`.

Reply:
Ahh good to know! 👍